README.md

```diff
@@ -1,167 +1,21 @@
-needs to run given a required service time.
-
-* The execution unit is approximately 100 `SQRTSD` x86 instructions.
-
-```bash
-$ kubectl apply -f server/benchmark/timing.yaml
-```
-
-Then, monitor and collect the `ITERATIONS_MULTIPLIER` from the job logs as follows:
-
-```bash
-$ watch kubectl logs timing
-```
-
-Finally, set the `ITERATIONS_MULTIPLIER` in the function template `workloads/$SANDBOX_TYPE/trace_func_go.yaml` to the
-value previously collected.
-
-To account for difference in CPU performance set `ITERATIONS_MULTIPLIER=102` if using
-Cloudlab `xl170` or `d430` machines. (Date of measurement: 18-Oct-2022)
-
-## Single execution
-
-To run load generator use the following command:
-
-```bash
-$ go run cmd/loader.go --config cmd/config.json
-```
-
-Additionally, one can specify log verbosity argument as `--verbosity [info, debug, trace]`. The default value is `info`.
-
-For to configure the workload for load generator, please refer to `docs/configuration.md`.
-
-There are a couple of constants that should not be exposed to the users. They can be examined and changed
-in `pkg/common/constants.go`.
-
-## Build the image for a synthetic function
-
-The reason for existence of Firecracker and container version is because of different ports for gRPC server. Firecracker
-uses port 50051, whereas containers are accessible via port 80.
-
-Synthetic function runs for a given time and allocates the required amount of memory:
-
-* `trace-firecracker` - for Firecracker
-* `trace-container` - for regular containers
-
-For testing cold start performance:
-
-* `empty-firecracker` - for Firecracker - returns immediately
-* `empty-container` - for regular containers - returns immediately
-
-```bash
-$ make build <trace-firecracker|trace-container|empty-firecracker|empty-container>
-```
-
-## Clean up between runs
-
-```bash
-$ make clean
-```
-
-## Running the Experiment Driver
-
-Within the tools/driver folder, Configure the driverConfig.json file based on your username on Cloudlab, 
-the address of the loader node that you are running the experiment on and any experiment parameters you'd like to set.
-Note that you need to have the relevant trace files on the machine running the experiment driver, they will then get
-transferred to the loader node.  
-Then run the following command to launch an experiment:
-```bash
-$ go run experiment_driver.go -c driverConfig.json
-```
-For more details take a look at the README in the tools/driver folder.
-
----
-
-For more options, please see the `Makefile`.
-
-## Using OpenWhisk
-
-For instructions on how to use the loader with OpenWhisk go to `openwhisk_setup/README.md`.
-
+# InVitro
+
+InVitro is a set of tools for conducting research in serverless computing. It consists of the sampler and the loader. The sampler is used for creating a representative workload summary from a real workload trace. The loader is a tool for generating the load based on a given trace, which is compatible with vHive and OpenWhisk.
+
```

Comment:
this doc needs to have the following subsections:
- license, etc (see vHive)
- Pointers to the quickstart and other first-order docs